pkgs/by-name/gi/git-lfs-transfer/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule {
+  pname = "git-lfs-transfer";
+  version = "0.1.0-unstable-2024-10-07";
+
+  src = fetchFromGitHub {
+    owner = "charmbracelet";
+    repo = "git-lfs-transfer";
+    rev = "422d24414fe4b803849b3f6fe7c4d8ab1b40803b";
+    hash = "sha256-YsplPW3i4W1RfkWQI1eGXFXb3JofQwKe+9LbjxeL1cM=";
+  };
+
+  vendorHash = "sha256-1cGlhLdnU6yTqzcB3J1cq3gawncbtdgkb3LFh2ZmXbM=";
+
```

Comment:
Would be good to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook if possible

Reply:
Unfortunately the binary has no --version